homeassistant/components/backup/strings.json

```diff
@@ -1,4 +1,14 @@
 {
+  "issues": {
+    "automatic_backup_failed_create": {
+      "title": "Automatic backup could not be created",
+      "description": "The automatic backup could not be created. Please check the logs for more information."
+    },
+    "automatic_backup_failed_upload_agents": {
+      "title": "Automatic backup could not be uploaded to agents",
+      "description": "The automatic backup could not be uploaded to agents {failed_agents}. Please check the logs for more information."
+    }
+  },
```

Comment:
Suggestions for improving the issue text are welcome

Reply:
Repair issues is not completely appropriate for this. We've said that they should be used for things that the user can address, and it's not certain that that is the case. We also don't tell the user what to do to solve it.

Reply:
Yeah, I agree, but this was discussed and it was decided to raise issues anyway to bring attention to the failed backup.